libnfc/buses/usbbus.h

```diff
@@ -35,6 +35,8 @@
 
 #ifndef _WIN32
 // Under POSIX system, we use libusb (>= 0.1.12)
+#define _GNU_SOURCE
```

Comment:
Defining feature test macros after headers have already been included is 
documented to be wrong, and can lead to nasty runtime breakages.